src/diffusers/pipelines/marigold/__init__.py

```diff
@@ -2,20 +2,40 @@
 
 from ...utils import (
     DIFFUSERS_SLOW_IMPORT,
+    OptionalDependencyNotAvailable,
```

Comment:
can you update this file?

Reply:
sorry I was confusing, I made a comment on your PR https://github.com/huggingface/diffusers/pull/7847/files#r1611782122